src/data/faq.json

```diff
@@ -58,9 +58,10 @@
-                            "The remaining encounters are rated collectively according to their distance and duration as well as the presumed infectiousness of the other person(s). This rating doesn't always yield an increased risk. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.", "For more details about risk assessment, including a sample calculation, see <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md' target='_blank' rel='noopener noreferrer'>https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md</a>."
+                            "All active Corona-Warn-Apps regularly download the diagnosis keys released on the Corona-Warn-App server and pass them on to the operating system in batches through an interface. The app checks whether any of these received, recorded rolling proximity identifiers match any of the diagnosis keys. If there is a match, this means the following: The user’s smartphone encountered the smartphone of a person who has uploaded a diagnosis key on the day to which the diagnosis key belongs.",
+                            "In the next step, the app analyzes all the matching rolling proximity identifiers for each diagnosis key, to estimate how long the exposure lasted in total on the day in question and how close the smartphones were to each other on average during the exposure. The distance is calculated from the measured reduction in strength of the Bluetooth signal, which is specified in dB (decibel). All exposures for a diagnosis key that lasted less than 10 minutes in total (regardless of how close the smartphones came during that time) or during which the smartphones were more than 8 meters (73 dB attenuation) apart on average (regardless of how long the exposure lasted) are rated as harmless. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.",
+                            "The remaining encounters are rated collectively according to their distance and duration as well as the presumed infectiousness of the other person(s). This rating doesn't always yield an increased risk. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.", 
```

Comment:
Nice!  
This change accurately describes the two ways how green encounters can happen 👍.
 
One thing I was wondering about, but unfortunately don't have a solution: The new entry now states twice in short succession 
> This is why the Corona-Warn-App can display encounters, but the risk status stays the same.

While this is o/c all correct, I was wondering whether we could formulate this a bit smoother, but I unfortunately don't have a suggestion right now. But it's also not really important, just some cosmetics. What do you think? 🙂